Source/ACE.DatLoader/DatReader.cs

```diff
@@ -93,16 +93,37 @@ public float ReadSingle()
             return data;
         }
 
+        /// <summary>
+        /// Returns a string as defined by the first byte's length
+        /// </summary>
         public string ReadPString()

```

Comment:
PString just seems to be their version of a basic string.  This seems to be identical to ReadString except it reads the length from the first byte.  I don't see any references to ReadString being used, so should this just replace ReadString?